static/app/views/insights/mobile/common/components/tables/screensTable.tsx

```diff
@@ -129,6 +147,7 @@ export function ScreensTable({
             key: columnKey,
             name: columnNameMap[columnKey],
             width: COL_WIDTH_UNDEFINED,
+            toolTip: columnTooltipMap ? columnTooltipMap[columnKey] : undefined,
```

Comment:
Instead of having to parse the tooltip out from the column tooltip map here, is it possible to just use the key in the `renderHeadCell` helper and pull the tooltip out from the map there? That way we can avoid having to edit the column type

We do want tooltips in headers one day so I think that's a good PR to have in the future, but for now this should work

Reply:
Oh right, yeah that should do the trick, thanks!